ndk-build/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+- Fix NativeActivity library path on non-UNIX systems, which resulted in runtime crashes on startup.
```

Comment:
Should probably make this a bit more clear than `resulted in runtime crashes on startup`. Something along the lines of: ``Pass UNIX path separators to `aapt` on non-UNIX systems, ensuring the resulting separator is treated as separator on Android instead of being part of the filename.``.

Not sure how to better word `being part of the filename` though, but there should be something more clear.